asset_stream_manager/BUILD

```diff
@@ -9,6 +9,7 @@ cc_binary(
     deps = [
         ":asset_stream_config",
         ":session_management_server",
+        "//cdc_stream",
```

Comment:
You're right, it's not necessary, it's just a convenience thing. Otherwise, I would have to add another VS project. My plan is to merge the two binaries into one, so that the service auto-starts when you start a client. But I can actually add the target in the vcxproj file, so I'll remove it here.